[2021-11-11 11:43:27,725] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2021-11-11T11:43:20.689342+00:00 [queued]>
[2021-11-11 11:43:27,897] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2021-11-11T11:43:20.689342+00:00 [queued]>
[2021-11-11 11:43:27,901] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-11 11:43:27,912] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-11 11:43:27,917] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-11 11:43:28,135] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): also_run_this> on 2021-11-11 11:43:20.689342+00:00
[2021-11-11 11:43:28,237] {standard_task_runner.py:52} INFO - Started process 1671 to run task
[2021-11-11 11:43:28,367] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'manual__2021-11-11T11:43:20.689342+00:00', '--job-id', '5', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp09a5cutw', '--error-file', '/tmp/tmpxu7408el']
[2021-11-11 11:43:28,384] {standard_task_runner.py:77} INFO - Job 5: Subtask also_run_this
[2021-11-11 11:43:28,880] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this manual__2021-11-11T11:43:20.689342+00:00 [running]> on host 150ca14c9dd9
[2021-11-11 11:43:29,631] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-11-11T11:43:20.689342+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-11T11:43:20.689342+00:00
[2021-11-11 11:43:29,644] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-11-11 11:43:29,658] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=manual__2021-11-11T11:43:20.689342+00:00 | dag_run=<DagRun example_bash_operator @ 2021-11-11 11:43:20.689342+00:00: manual__2021-11-11T11:43:20.689342+00:00, externally triggered: True>"']
[2021-11-11 11:43:29,781] {subprocess.py:85} INFO - Output:
[2021-11-11 11:43:29,821] {subprocess.py:89} INFO - run_id=manual__2021-11-11T11:43:20.689342+00:00 | dag_run=<DagRun example_bash_operator @ 2021-11-11 11:43:20.689342+00:00: manual__2021-11-11T11:43:20.689342+00:00, externally triggered: True>
[2021-11-11 11:43:29,822] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-11 11:43:30,053] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20211111T114320, start_date=20211111T114327, end_date=20211111T114330
[2021-11-11 11:43:30,255] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-11 11:43:30,674] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
